packages/scss/src/components/switch/component.scss

```diff
@@ -37,7 +37,7 @@
 				position: absolute;
 				top: var(--spacings-smallest);
 				transition: all var(--commons-animations-durations-fast) ease;
-				width: calc(2rem - 4px);
+				width: calc(var(--spacings-big) - 4px);
```

Comment:
Why not `- var(--spacings-smallest)` instead of `- 4px` following your previous logic?

Reply:
I don't know why we have a pixel value, my work here was only for standard replacements.